acq4/devices/Camera/Camera.py

```diff
@@ -2,6 +2,8 @@
 from __future__ import with_statement
 from acq4.devices.DAQGeneric import DAQGeneric, DAQGenericTask
 from acq4.devices.OptomechDevice import OptomechDevice
+from acq4.devices.LightSource import LightSource
```

Comment:
Luke...Thanks for the feedback.  I'll go through all this and update with the changes first thing on monday morning.